lib/pebbles/pebble_controller.flow

```diff
@@ -49,6 +49,9 @@ export {
 	restoreLastPebble(controller : PebbleController, pebble : Pebble) -> void;
 	// Just refresh the view, i.e. call dispatcher once again. No URL or history changes 
 	refreshViewForCurrentPebble : (controller : PebbleController) -> void;
+	// we will find last visited pebble by this key
+	// in case of js and onJS is true - last visited pebble will be open at start without prompt
+	setLastPebbleKey(key : string, onJS : bool) -> void;
```

Comment:
Explain how to use this function in the app. Also mentioned that it will be saved by default in CPP case, and `onJS` flag is a way to control it in JS case. Also I would rename "onJS" to "saveLastPebbleInJsCase" or something like this